cs-amazon/src/main/java/io/cloudslang/content/amazon/entities/aws/Attribute.java

```diff
@@ -29,7 +29,7 @@
         this.value = value;
     }
 
-    public String getValue() {
+    private String getValue() {
```

Comment:
@gituser173 Regarding consistency: both Attribute and InstanceFilter classes have getValue as the rest of entities... but private. Made them private because those methods are used only inside the enums. If (in the future) there will be needed then we can raise them to be public.  Howevere both enums have the method getAttribute/getFilter for 2 reasons: 1. to return the same "Not relevant" string in case the filter is not set; 2. to validate input... So, IMHO is ok as they are right now...